### ISSUE TITLE
Running `ember serve` from Intellij IDE through `Run` (^R) throws errors - existing with code 127

### ISSUE THREAD

**Issue by rinoldsimon** [Turbo87/intellij-emberjs#238]
### Description

/Users/workspace/web_ui/node_modules/.bin/ember serve
env: node: No such file or directory

Process finished with exit code 127

### Steps to Reproduce

- at times, after newly opening IntelliJ IDE
- when i restart the IDE it works fine

**Expected behavior:** Start ember server

**Actual behavior:** Throws error in `Run` tool window

**Reproduces how often:** Alternate number of times.

### Versions

- IntelliJ:

IntelliJ IDEA 2018.2.5 (Ultimate Edition)
Build #IU-182.4892.20, built on October 16, 2018

- The version of `intellij-emberjs`:

2018.2.4-0

**Comment by Turbo87** [comment]
it sounds like IntelliJ can't find your Node.js executable

**Comment by rinoldsimon** [comment]
why does it happen at alternate number of time? However i specified node path under 
`preference -> languages & frameworks -> node.js and npm`

**Comment by Turbo87** [comment]
> why does it happen at alternate number of time?

I don't know...

> However i specified node path under
> `preference -> languages & frameworks -> node.js and npm`

I think we might not use that path but instead look for a global `node` binary. @makepanic might know this?

**Comment by makepanic** [comment]
Yes we execute the `node_modules/.bin/ember` executable directly.
I guess on your system `#!/usr/bin/env node` returns nothing.

Maybe it's a good idea to also check if we can access the global node binary.
